backend/src/main/java/pl/krkteam/battleships/room/holder/dto/create/room/CreateResultDTO.java

```diff
@@ -2,4 +2,8 @@
 
 public abstract class CreateResultDTO {
     String result;
```

Comment:
private

Reply:
it's abstract class, so I that field would be private I don't have access for extending classes to it